skinning/skinning_test_simd.cpp

```diff
@@ -162,7 +162,7 @@ void calculateVerticesAndNormals_SSE_intrinsics(
       _mm_storeh_pi((__m64*)output_vertex, sum2);
 
       __m128 sum3 = _mm_shuffle_ps(sum2, sum2, _MM_SHUFFLE(1, 1, 1, 1));
-      sum3 = _mm_add_ss(sum2, sum3);
+      sum3 = _mm_add_ps(sum2, sum3);
```

Comment:
For example, in my test, with emscripten patch (https://github.com/kripken/emscripten/pull/3120), the number of _mm_add_ss is:
````
/home/nhu/devel/simd.js/v8-m40/out/ia32.release/d8 --simd-object emscripten-O3-simd.js
Skinned vertices per second: 83608000, blah=0.000000
````
The number of _mm_add_ps is:
```
/home/nhu/devel/simd.js/v8-m40/out/ia32.release/d8 --simd-object emscripten-O3-simd.js
Skinned vertices per second: 89698000, blah=0.000000
```